Sources/StreamChatSwiftUI/ChatChannel/MessageList/DeletedMessageView.swift

```diff
@@ -28,14 +29,18 @@ public struct DeletedMessageView: View {
                 .standardPadding()
                 .foregroundColor(Color(colors.textLowEmphasis))
                 .messageBubble(for: message, isFirst: isFirst)
+            
             HStack {
                 Spacer()
-                Image(systemName: "eye")
+                
+                Image(uiImage: images.eye)
```

Comment:
The eye icon is still black, it should be grey like in the current version. I think we need to explicitly provide the color in this case.

Reply:
Fixed through the usage of the custom modifier!